include/stag/EDInterface.h

```diff
@@ -14,6 +14,9 @@ class EDInterface
 	EDLines* edLines = NULL;
 
 public:
+
+	~EDInterface();
```

Comment:
```suggestion
	~EDInterface() {
		delete edgeMap;
		delete edLines;
	}
```
This was how [ManfredStoiber](https://github.com/ManfredStoiber/stag/commit/e40fb04b1820e97b78b339f6bdca8a53e33e1171#diff-2eb588e86910910523de48a59c93a3c6e4333812409593c4b7b5fa9f49025117R16) defined their destructor. Do you think it's better to be explicit in what gets deleted too?

Reply:
This would follow how you destructed EDLines